services/business/services.go

```diff
@@ -28,61 +24,32 @@ func (in *SvcService) GetServiceList(namespace, rateInterval string) (*models.Se
 	if err != nil {
 		return nil, err
 	}
-
-	// Fetch services requests rates
-	inRates, outRates, _ := in.prom.GetNamespaceServicesRequestRates(namespace, rateInterval)
+	nsHealth := in.health.getNamespaceHealth(namespace, kubernetesServices, rateInterval)
```

Comment:
I thought the goal was to separate the health from the GetServiceList to perform two async calls.
But here I see it's also fetched.
Perhaps I missed some detail.

Reply:
No, not for this task. Kiali-911 is about graph only. I think there's another one to change ServiceList behaviour